src/main/kotlin/no/nav/omsorgspenger/infotrygd/InfotrygdRamme.kt

```diff
@@ -69,9 +69,15 @@ internal data class InfotrygdFordelingGirMelding(
 internal data class InfotrygdOverføringFårMelding(
     override val periode: Periode,
     override val kilder: Set<Kilde>,
+    val dato: LocalDate, // // todo: er denne på rett plass? Den returneres fra infotrygd-grunnlag-paaroerende-sykodm og jeg antar at denne tilsvarer "gjennomført"
+    val annenPart: InfotrygdAnnenPart, // todo: er denne på rett plass?
```

Comment:
Ja, bare kall den for `barn` her - ellers kan nok objektet mappes ganske direkte fra responsen;
https://github.com/navikt/omsorgspenger-infotrygd-rammevedtak/blob/559b8eba2b289bcd01fbe9007a2099262f4dae2f/src/test/kotlin/no/nav/omsorgspenger/api/HenteRammevedtakTest.kt#L84